source/query-bar.txt

```diff
@@ -57,20 +57,17 @@ operator.
    Builder to enter a query into the query bar. For details, see
    :ref:`build-query`.
 
-As you type, the :guilabel:`Apply` button is disabled and the
+As you type, the :guilabel:`Find` button is disabled and the
 :guilabel:`Filter` label turns red until a valid query is entered.
 Filter syntax requires strict :manual:`extended JSON
-</reference/mongodb-extended-json/>` types. For instance, to specify an
+</reference/mongodb-extended-json/>` types. To specify an
 ``ObjectID``, use the format ``{ "$oid": "<string>" }``
```

Comment:
I don't think the text disagrees with that assertion. Not sure what your point is, but I added a link to ObjecttId.